core/exception/src/main/java/uk/gov/gchq/gaffer/core/exception/ErrorFactory.java

```diff
@@ -60,11 +59,10 @@ public static Error from(final GafferCheckedException gex) {
      */
     public static Error from(final GafferRuntimeException gex) {
         LOGGER.error("Error: {}", gex.getMessage(), gex);
-        return new ErrorBuilder()
-                .status(gex.getStatus())
-                .simpleMessage(gex.getMessage())
-                .detailMessage(ExceptionUtils.getStackTrace(gex))
-                .build();
+        return new ErrorBuilder().status(gex.getStatus())
```

Comment:
Please can we leave this formatting as it was? It would help the review have less changes and I think it looks neater having each method call on a new line.